### PR TITLE
adding image classification workshop notebook

### PR DESCRIPTION
**Pull request by mchestnut91**
Hi Ben,

This PR contains the workshop that I put together for class on Saturday. In this workshop students will use sckit-learn's digits dataset to predict digits using Logistic Regression and SVM. The main purpose of this workshop is to review concepts related to cross-validation and grid search.

I ran all of the cells to help you go through the notebook quickly, but I plan to remove some of the code so the students will have to call some of the functions themselves in an attempt to make it more interactive. Let me know what you think!

Best regards,
-Michael

### CODE REVIEW

File: notebook/image-classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,474 @@
+{
```
Comment (bbengfort) [inline notebook/image-classification.ipynb]: <p>Perhaps you could use the <code>gray_r</code> (gray reversed) cmap instead to make this a black/gray number on a white background?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgetown-analytics/machine-learning/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Comment by bbengfort** [comment]
Looks great to me! Do you want me to merge this in now and you can do any followups you'd like as a PR or would you like me to keep this PR open so you can make changes?

File: notebook/image-classification.ipynb (hunk 1 above)
Comment (mchestnut91) [reply notebook/image-classification.ipynb]: Thanks for the tip! I think that would look better. You should start a visualization library or something..

**Comment by mchestnut91** [comment]
Thank you! Go ahead and merge it for now, I'll submit another PR to clean it up and have it ready for Saturday.

**Pull request merged**